internal/provider/resource_integration_test.go

```diff
@@ -250,3 +251,150 @@ resource "juju_integration" "a" {
 }
 `, srcModelName, dstModelName, viaCIDRs)
 }
+
+func TestAcc_ResourceIntegrationWithMultipleConsumers_Stable(t *testing.T) {
+        if testingCloud != LXDCloudTesting {
+                t.Skip(t.Name() + " only runs with LXD")
+        }
+        srcModelName := acctest.RandomWithPrefix("tf-test-integration")
+        dstModelName := acctest.RandomWithPrefix("tf-test-integration-dst")
+
+        resource.Test(t, resource.TestCase{
+                PreCheck: func() { testAccPreCheck(t) },
+                ExternalProviders: map[string]resource.ExternalProvider{
+                        "juju": {
+                                VersionConstraint: TestProviderStableVersion,
+                                Source:            "juju/juju",
+                        },
+                },
```

Comment:
This means that the test is running against 0.8.0, not the code being changed. Use ProtoV6ProviderFactories instead.

```suggestion
                ProtoV6ProviderFactories: frameworkProviderFactories,
```

Reply:
Modified as per the suggesion